packages/phone-number-privacy/combiner/src/config.ts

```diff
@@ -1,7 +1,7 @@
 import { OdisUtils } from '@celo/identity'
 import { BlockchainConfig, rootLogger, TestUtils, toBool } from '@celo/phone-number-privacy-common'
 import * as functions from 'firebase-functions'
-export const VERSION = process.env.npm_package_version ?? ''
+export const VERSION = process.env.npm_package_version ?? '0.0.0'
```

Comment:
No objection but curious why this change was made

Reply:
it's to match the [default value in the signer](https://github.com/celo-org/celo-monorepo/blob/8ac130841547a86e6f14782c7a1eded39eacd6e1/packages/phone-number-privacy/signer/src/config.ts#L7), as the mismatch was causing the tests to fail